spec/fixtures/iso-sample-identifiers-v2.txt

```diff
@@ -5848,7 +5850,7 @@ ISO 8969:2011
 ISO 9073-11:2004
 ISO 1143:2010
 ISO 14708-7
-ISO /IEC 17030
+ISO/IEC 17030
```

Comment:
I wonder if we can be more lenient with these missing spaces? e.g. missing space between originator and number, missing space between originator and co-publisher...